starfish/intensity_table/intensity_table_coordiantes.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import xarray as xr
+
+from starfish.imagestack.imagestack import ImageStack
+from starfish.imagestack.physical_coordinate_calculator import get_physcial_coordinates_of_spot
+from starfish.intensity_table.intensity_table import IntensityTable
+from starfish.types import Coordinates, Features, Indices
+
+
+def transfer_physical_coords_from_imagestack_to_intensity_table(image_stack: ImageStack,
+                                                                intensity_table: IntensityTable
+                                                                ) -> IntensityTable:
+    """
+    Transfers physical coordinates from an Imagestack's coordinates xarray to an intensity table
+        1.) Creates three new coords on the intensity table (xc, yc, zc)
+        2.) For every spot:
+                Get pixel x,y values
+                Calculate the physical x,y values
+                Assign those values to the coords arrays for this spot
+    """""
+    # Add three new coords to xarray (xc, yc, zc)
+    num_features = intensity_table.sizes[Features.AXIS]
+    intensity_table[Coordinates.X.value] = xr.DataArray(np.zeros(num_features, np.float32),
+                                                        dims='features')
```

Comment:
out of curiosity, does `dims='features'` affect an xarray coordinate?

Reply:
Yea I ran into this issue from xarray that was like "coordinates can not have new dimensions" because the default dimension getting added didn't match the 'features' dim on the intenisty table, this seemed to be the fix.